src/Facebook/Facebook.php

```diff
@@ -327,7 +350,7 @@ public function getRedirectLoginHelper()
    */
   public function getJavaScriptHelper()
   {
-    return new FacebookJavaScriptHelper($this->app);
+    return new FacebookJavaScriptHelper($this->app, $this->getOAuth2Client());
```

Comment:
All of these helpers extend from the `FacebookSignedRequestFromInputHelper` which needs an instance of `FacebookApp` to inject to the `SignedRequest` entity. I considered pulling the `FacebookApp` from the `OAuth2Client` within `FacebookSignedRequestFromInputHelper`, but it felt wrong since signed requests and OAuth flow are two separate concepts that shouldn't depend on each other.